app/models/comment.rb

```diff
@@ -0,0 +1,2 @@
+class Comment < ApplicationRecord
```

Comment:
This could include the associations between post and comments: "belongs_to posts" for example.